<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_ABBAddr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ABBPort">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ServerAddr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ServerPort">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ABBIP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ServIP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditPos_x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPos_y">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPos_z">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRun">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>RUN</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonGo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionauto_while_balance"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionconnect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenCamera">
   <property name="text">
    <string>OpenCamera</string>
   </property>
  </action>
  <action name="actionOpenCamera_2">
   <property name="text">
    <string>OpenCamera</string>
   </property>
  </action>
  <action name="actionOpenCamera_3">
   <property name="text">
    <string>OpenCamera</string>
   </property>
  </action>
  <action name="actionauto_while_balance">
   <property name="text">
    <string>auto while balance</string>
   </property>
  </action>
  <action name="actionconnect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
